### PR TITLE
test: Updated test for list widget

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
Scenario's covered

- Widgets within list widget have no dynamic height
- Widgets within list widget in existing applications have no dynamic height
- Widgets when moved out of the list widget have dynamic height in property pane
- Widgets when moved into the list widget have no dynamic height
- Widgets which have dynamic height outside the list widget when moved into the list widget should no longer have dynamic height
- Widgets when copied and pasted into the list widget no longer have dynamic height
- Widgets when cut and pasted into the list widget no longer have dynamic height
- Widgets which were not possible to be added to list widget cannot be pasted/moved into the list widget.

### CODE REVIEW

File: app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/DynamicHeight/DynamicHeight_List_TextWidget_Spec.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+describe("Dynamic Height Width validation list widget", function () {
+    afterEach(() => {
+        agHelper.SaveLocalStorageCache();
+    });
+
+    beforeEach(() => {
+        agHelper.RestoreLocalStorageCache();
+    });
+    it("Validate change with auto height width for list widgets", function () {
```
Comment (riodeuno) [inline app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/DynamicHeight/DynamicHeight_List_TextWidget_Spec.js:16]: @NandanAnantharamu Let's add comments here where we describe what we're checking. The code is difficult to read for me.

**Review by riodeuno — CHANGES_REQUESTED** [review]

File: app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/DynamicHeight/DynamicHeight_List_TextWidget_Spec.js (hunk 1 above)
Comment (NandanAnantharamu) [reply app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/DynamicHeight/DynamicHeight_List_TextWidget_Spec.js:16]: I have added comments ,one thing drag and drop usecase were not possible so we have added tests using copy paste feature

**Review by rahulramesha — APPROVED** [review]

**Review by riodeuno — APPROVED** [review]

**Pull request merged**